src/Symfony/Component/Security/Core/Resources/translations/security.ru.xlf

```diff
@@ -70,6 +70,14 @@
                 <source>Invalid or expired login link.</source>
                 <target>Ссылка для входа недействительна или просрочена.</target>
             </trans-unit>
+            <trans-unit id="19">
+                <source>Too many failed login attempts, please try again in %minutes% minute.</source>
```

Comment:
Am I right that the first unit is used for singular form only and the second one for all the rest?

Reply:
yes.

https://github.com/symfony/symfony/blob/bb1e1e58aea5318e96d1c22cc8a91668ed7baaaa/src/Symfony/Component/Security/Core/Exception/TooManyLoginAttemptsAuthenticationException.php#L44